File: Projects/MYONN/1_Gentle_Start_with_Python/func_avg.py
#
# Gentle start of Python
#   Function that takes 2 numbers as input
#   and outputs their average
# Filename: func_avg.py
#

def avg(x,y):
    print("first input is: ", x)
    print("second input is: ", y)
    a = (x+y)/2.0
    print("average is ", a)
    return a

if __name__=="__main__" :
    x = 3
    y = 5
    print(F"Average of {x} and {y} is {avg(x,y)}")

File: Projects/MYONN/1_Gentle_Start_with_C++/func_avg.cpp
//
// Gentle start of C++
// Filename: func_avg.cpp
//
#include <stdio.h>
#include <stdlib.h>

float avg(float x, float y)
{
    printf("first input is %f\n", x);
    printf("second input is %f\n", y);

    float a = (x+y)/2.0;
    printf("average is %f\n", a);

    return a;
}

int main()
{
    printf("Function avg() returns %f\n", avg(3.2, 7.5));
}
